site/static/examples/kind-with-registry.sh

```diff
@@ -7,15 +7,15 @@ KIND_CLUSTER_NAME="${KIND_CLUSTER_NAME:-kind}"
 # create registry container unless it already exists
 reg_name='kind-registry'
 reg_port='5000'
-running="$(docker inspect -f '{{.State.Running}}' "${reg_name}")"
+running="$(docker inspect -f '{{.State.Running}}' "${reg_name}" 2>/dev/null || :)"
 if [ "${running}" != 'true' ]; then
   docker run \
     -d --restart=always -p "${reg_port}:5000" --name "${reg_name}" \
     registry:2
 fi
 
 # create a cluster with the local registry enabled in containerd
-cat <<EOF | kind create cluster --config=-
+cat <<EOF | kind create cluster --name "${reg_name}" --config=-
```

Comment:
NACK on this change, `${reg_name}` is the name of the registry container, NOT the cluster.

you can avoid collosion by:
- setting `KIND_CLUSTER_NAME`
- modifying the script

Reply:
I see now, my understanding of `KIND_CLUSTER_NAME` was incorrect. I though it was a secondary cluster created just for the purpose of demonstrating the registry.